volume/errors.go

```diff
@@ -0,0 +1,24 @@
+package volume
+
+// Custom credential operation error codes
+const (
+	_ = iota + 100
+	// ErrInvalidCredential is returned when a credential is invalid
+	ErrInvalidCredential
```

Comment:
I don't think we need Code, since we don't know what failed when validating. The message has enough info on what went wrong

Reply:
Was just trying to make it future proof, where we should ideally be sending specific error codes. For now I will switch to grpc status instead of introducing a new error.

Reply:
I don't think we should use grpc errors for the REST APIs

Reply:
Okay, will keep it as is and remove the code.